### ISSUE TITLE
swipebox is not working

### ISSUE THREAD

**Issue by manusharma1** [micahalcorn/meteor-justified-gallery#1]
Hi,

I am using the justified gallery everything is working except the swipebox.

I have included the swipebox js and css files into the meteorapp > client > lib folders 

Template

```
        <div id="swipeboxExample">
      {{#each theFiles}}
        <a href="{{url}}"><img alt="title" src="{{url}}" /></a>
      {{/each}}
    </div>
```

Helper

```
Template.newhomepage7.helpers({
  theFiles: function () {
return ImageFileCollection.find();
  }
});
```

On Render

```
     $('#swipeboxExample').justifiedGallery({
lastRow : 'nojustify', 
rowHeight : 100, 
rel : 'gallery2',
margins : 1
}).on('jg.complete', function () {
    console.log('reached here');
$('#swipeboxExample a').swipebox();
});
}
```

Even the console is displaying me the log as  "reached here"

there is no error that I can see in the log

Please help.

Thanks and Regards,
Manu

**Comment by micahalcorn** [comment]
Sorry, I was not aware of Swipebox when I built this package. It is now included and I'm going to be using it in my project in place of the other solution that I had. :) I also updated the Justified Gallery plugin to the latest version. Let me know if you have any other issues after running `meteor update micah:justified-gallery`.

Resolved by 384986052bffafee4d5827743a2277b35fcccbf5.

**Comment by manusharma1** [comment]
Thanks for the Update
